Cmdline/CKAN-cmdline.csproj

```diff
@@ -9,12 +9,15 @@
     <AssemblyName>CmdLine</AssemblyName>
     <StartupObject>CKAN.CmdLine.MainClass</StartupObject>
     <ApplicationIcon>..\GUI\assets\ckan.ico</ApplicationIcon>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
```

Comment:
I'm guessing we didn't have a target framework before? What would it default to without this? (My mind says that we were targeting dotnet40, but I've no idea if that's actually true.)